packages/utilities/src/types/ManagerPreferences.ts

```diff
@@ -22,6 +22,7 @@ export type ThemeChoice = 'dark' | 'light' | 'system';
 
 export type ManagerPreferences = Partial<{
   aclpPreference: AclpConfig; // Why is this type in @linode/api-v4?
+  alertsGroupByTag: boolean;
```

Comment:
```suggestion
  aclpAlertsGroupByTag: boolean;
```

Because there is already some existing alerts in linode...lets put aclp here